### PR TITLE
only create table if exists, clean entries

### PR DESCRIPTION
**Pull request by vaikas**

### CODE REVIEW

File: cmd/initdb/main.go [hunk 1]
```diff
@@ -25,27 +26,58 @@ var entries = []Entry{
...
 const fmtStr = "insert into users (first_name, last_name, email) values ('%s', '%s', '%s')"
+const tableQuery = `SELECT tablename FROM pg_catalog.pg_tables where tablename = $1`
+
+func checkTableExists(db *sql.DB) (bool, error) {
+	rows, err := db.Query(tableQuery, "users")
+	if err != nil {
+		return false, err
+	}
+	defer rows.Close()
+	for rows.Next() {
```
Comment (mattmoor) [inline cmd/initdb/main.go:37]: Do you have to walk through the rows or can you just close?

File: cmd/initdb/main.go [hunk 2]
```diff
@@ -25,27 +26,58 @@ var entries = []Entry{
...
-	if err != nil {
-		log.Fatalf("Failed to create table users: %v", err)
+		if err != nil {
+			log.Fatalf("Failed to create table users: %v", err)
+		}
+		log.Printf("Created Table: %+v", res)
 	}
 
-	log.Printf("Created Table: %+v", res)
+	_, err = db.Exec("delete from users")
```
Comment (mattmoor) [inline cmd/initdb/main.go:74]: nit: capitalize keywords 
```suggestion
	_, err = db.Exec("DELETE FROM users")
```

File: cmd/initdb/main.go [hunk 3]
```diff
@@ -25,27 +26,58 @@ var entries = []Entry{
 }
 
 const fmtStr = "insert into users (first_name, last_name, email) values ('%s', '%s', '%s')"
+const tableQuery = `SELECT tablename FROM pg_catalog.pg_tables where tablename = $1`
```
Comment (mattmoor) [inline cmd/initdb/main.go:29]:
```suggestion
const tableQuery = `SELECT tablename FROM pg_catalog.pg_tables WHERE tablename = $1`
```

File: cmd/initdb/main.go (hunk 1 above)
Comment (vaikas) [reply cmd/initdb/main.go:37]: it could return no rows and it wouldn't be an error, so I think so?

**Pull request merged**